manifests/config.pp

```diff
@@ -92,7 +97,7 @@
     ensure => 'directory',
     owner  => 'root',
     group  => $auditd::log_group,
-    mode   => 'o-rwx'
+    mode   => $log_dir_mode,
```

Comment:
This is the only change that will make a difference from the original implementation.

The intent here was to leave the group permissions alone in case ACLs had been set specifically to allow read access.

What are your thoughts on changing this to `g+rX,o-rwx` if the user is not `root`? We should probably also make this recursive so that the underlying files are changed for reading.

Reply:
@trevor-vaughan Does this look correct?

Reply:
@coreone You can actually get rid of `log_dir_mode` and switch them all back over to `log_file_mode`. Puppet does "the right thing" under the hood for directories.

You'll also need to add `recurse => true` to the `/var/log/audit` resource so that the files are appropriately modified if there's an after-the-fact switch.

Reply:
@coreone If you're OK with it, I can make the changes real quick  and get this wrapped up.

Reply:
If that is set to `recurse`, the next stanza should probably be removed I assume as it should inherit permissions on the log file itself instead of being manually set:

```puppet
file { $auditd::log_file:
  owner => 'root',
  group => $auditd::log_group,
  mode  => $log_file_mode
}
```

Reply:
Yeah, it would be redundant at that point. Good catch.

Reply:
If you want to make any final changes, I'm fine with that. Was just looking through the spec tests and some of those would need to be updated as well.

Reply:
@coreone Created a PR against your fork